cishouseholds/edit.py

```diff
@@ -124,7 +134,26 @@ def clean_postcode(df: DataFrame, postcode_column: str):
     return df.drop("TEMP")
 
 
-def update_from_csv_lookup(df: DataFrame, csv_filepath: str, id_column: str):
+def update_cohort_variable(
+    df: DataFrame, csv_filepath: str, current_cohort: str, study_cohort: str, participant_id: str
+):
+    """
+    Update cohort variable from seprate csv
+    """
+    spark = get_or_create_spark_session()
+    csv = spark.read.csv(csv_filepath, header=True)
+    df = df.join(csv, on=[participant_id, current_cohort], how="left")
```

Comment:
This won't run with the test data, as the name of the column isn't the same on both dataframes. The lookup has an old and new equivalent.